assets/js/util/when-active.js

```diff
@@ -45,7 +46,7 @@ const { useSelect } = Data;
  * @param {WPComponent|null} [options.IncompleteComponent] Optional. Fallback component to render when the module is active but not connected.
  * @return {Function} Enhancing function.
  */
-export default function whenActive( { moduleName, FallbackComponent = null, IncompleteComponent = null } ) {
+export default function whenActive( { moduleName, FallbackComponent = WidgetNull, IncompleteComponent = null } ) {
```

Comment:
We should rely on `WidgetNull` from props everywhere except for within `getWidgetComponentProps`. Since this is an HOC for use with widgets, we can assume it will be provided here too.
```suggestion
export default function whenActive( {
	moduleName,
	WidgetNull,
	FallbackComponent = WidgetNull,
	IncompleteComponent = null,
} ) {
```